bin/out.js

```diff
@@ -41,10 +41,28 @@ stdin.on('end', function () {
     processWebhook(source, params);
 });
 
+function buildUrl(source, params) {
+    const instanceVars = buildInstanceVariables();
+    return encodeURI(`${env.ATC_EXTERNAL_URL}/api/v1/teams/${env.BUILD_TEAM_NAME}/pipelines/${env.BUILD_PIPELINE_NAME}/resources/${params.resource_name}/check/webhook?webhook_token=${params.webhook_token}${instanceVars}`);
+}
+
+function buildInstanceVariables() {
+    let vars = "";
+    try {
+        const instanceVars = JSON.parse(env.BUILD_PIPELINE_INSTANCE_VARS)
+        for (const [key, value] of Object.entries(instanceVars)) {
+            vars += `&vars.${key}="${value}"`;
+        }
+    } catch(exception) {
+        log(`Exception building instance variable: ${exception}`)
```

Comment:
If an exception occurs, it would be better to fail the script and let the user know that they provided invalid JSON instead of creating a webhook with a configuration they probably didn't want.